tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -131,7 +131,9 @@ def test_activate_single_service(super_client, admin_client, sim_context, nsp):
         list_serviceExposeMap(serviceId=service.id)
 
     assert len(instance_service_map) == 1
-    assert instance_service_map[0].state == 'active'
+    wait_for_condition(
```

Comment:
just curious, we do you use this long form? does `x = client.wait_success(x)` not work?